### DISCUSSION TITLE
在原有的vue3项目上使用Xflow该如何配置

### DISCUSSION THREAD

**Discussion by qiaofenlin** [antvis/X6#4013] [Q&A]
在原有的vue3项目上使用Xflow该如何配置

**Comment by lloydzhou** [comment]
xflow是react写的
意思是你要在一个vue项目中集成react的项目
如果一定要这么做，你可以试试微前端框架，例如 [qiankun](https://qiankun.umijs.org/zh/guide)

**Comment by lloydzhou** [comment]
其实你可以看看 awesome-x6这边收集的几个vue的项目
https://github.com/lloydzhou/awesome-x6#vue-component

我比较推荐使用这两个在vue中集成x6
![image](https://github.com/antvis/X6/assets/1826685/9fe3f187-ef0e-49cd-b2b8-b998589e175e)
